### PR TITLE
Change == to ===

### PR DESCRIPTION
**Pull request by arv**
Also changes != to !==.

Motivation: == and != are disallowed in strong mode.

### CODE REVIEW

**Comment by arv** [comment]
@johnjbarton PTAL

File: build/build-unicode-tables.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 var http = require('http');
 var util = require('./util.js');
-var print = console.log.bind(console);
+var print = util.print;
```
Comment (arv) [inline build/build-unicode-tables.js]: This change is just to reduce the diffs between the two branches.

File: src/Options.js [hunk 2]
```diff
@@ -42,8 +42,8 @@ export var optionsV01 = enumerableOnlyObject({
   freeVariableChecker: false,
   generatorComprehension: false,
   generators: true,
-  lowResolutionSourceMap: false,
   inputSourceMap: false,
+  lowResolutionSourceMap: false,
```
Comment (arv) [inline src/Options.js]: This too.

**Comment by johnjbarton** [comment]
LGTM

**Pull request closed without merging**